GitUI/CommandsDialogs/SettingsDialog/Pages/AppearanceFontsSettingsPage.Designer.cs

```diff
@@ -66,20 +69,23 @@ private void InitializeComponent()
             this.tableLayoutPanel1.ColumnStyles.Add(new System.Windows.Forms.ColumnStyle());
             this.tableLayoutPanel1.ColumnStyles.Add(new System.Windows.Forms.ColumnStyle(System.Windows.Forms.SizeType.Percent, 100F));
             this.tableLayoutPanel1.Controls.Add(this.label56, 0, 0);
+            this.tableLayoutPanel1.Controls.Add(this.monospaceFontChangeButton, 1, 3);
             this.tableLayoutPanel1.Controls.Add(this.commitFontChangeButton, 1, 2);
             this.tableLayoutPanel1.Controls.Add(this.diffFontChangeButton, 1, 0);
             this.tableLayoutPanel1.Controls.Add(this.applicationFontChangeButton, 1, 1);
+            this.tableLayoutPanel1.Controls.Add(this.label36, 0, 3);
             this.tableLayoutPanel1.Controls.Add(this.label34, 0, 2);
             this.tableLayoutPanel1.Controls.Add(this.label26, 0, 1);
             this.tableLayoutPanel1.Dock = System.Windows.Forms.DockStyle.Fill;
             this.tableLayoutPanel1.Location = new System.Drawing.Point(8, 22);
             this.tableLayoutPanel1.Name = "tableLayoutPanel1";
-            this.tableLayoutPanel1.RowCount = 3;
+            this.tableLayoutPanel1.RowCount = 4;
             this.tableLayoutPanel1.RowStyles.Add(new System.Windows.Forms.RowStyle());
             this.tableLayoutPanel1.RowStyles.Add(new System.Windows.Forms.RowStyle());
             this.tableLayoutPanel1.RowStyles.Add(new System.Windows.Forms.RowStyle());
-            this.tableLayoutPanel1.Size = new System.Drawing.Size(2116, 87);
-            this.tableLayoutPanel1.TabIndex = 6;
+            this.tableLayoutPanel1.RowStyles.Add(new System.Windows.Forms.RowStyle());
+            this.tableLayoutPanel1.Size = new System.Drawing.Size(2116, 116);
+            this.tableLayoutPanel1.TabIndex = 7;
```

Comment:
Could you please ensure tab indicies are still in the logical order

Reply:
They are, I moved `this.tableLayoutPanel1.TabIndex = 7;` +1 up to make space for `this.monospaceFontChangeButton.TabIndex = 6;`
When tabbing inside the form focus is moving as expected and in the same order as before taking into account the new button.

By the way... this tableLayoutPanel1 thing, I found it impossible to add controlls into it through the designer. Every single change in `.Designer.cs` had to be done in notepad++ as VS reorders everything completely on save. Is that.... is that how it's supposed to be? :/

Reply:
> Every single change in .Designer.cs had to be done in notepad++ as VS reorders everything completely on save. Is that.... is that how it's supposed to be?

The designer has its own mind how to format a file and this often happens You can edit the Designer cs file in VS too.

Reply:
I have (I think) `Productivity Power Tools` doing `Format Document On Save` which does a gazillion tiny things to `.Designer.cs` files which is why I chose notepad++ rather than changing settings.

Reply:
I make changes in the designer, but then often cherry pick and stage selected lines.